apps/condo/domains/acquiring/access/PaymentsFilterTemplate.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Generated by `createschema acquiring.PaymentsFilterTemplate 'name:Text; employee:Relationship:OrganizationEmployee:CASCADE'`
+ */
+
+const { throwAuthenticationError } = require('@condo/domains/common/utils/apolloErrorFormatter')
+const { queryOrganizationEmployeeFor } = require('@condo/domains/organization/utils/accessSchema')
+const { getById, getByCondition } = require('@core/keystone/schema')
+
+async function canReadPaymentsFilterTemplates ({ authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+
+    if (user.isAdmin) return {}
+
+    return {
+        employee: { organization: { ...queryOrganizationEmployeeFor(user.id) } },
```

Comment:
why related organization in Export access and no related-org logic here?

Reply:
Here is the same decision as for templates for tickets filtering.
Anyway, this is a business related decision: should we give access to templates for employees from related organizations or not.

Reply:
You right. This security issue fixed.